mysql-test/t/percona_userstat_bug728082.test

```diff
@@ -0,0 +1,43 @@
+--source include/have_innodb.inc
```

Comment:
Please do not introduce new test file, but extend the existing one - this is a basic test for this particular column